travis.sh

```diff
@@ -11,8 +12,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-codecov:
-  branch: master
-comment:
-  branches:
-  - master
+
+set -e
+set -x
+# Set pipefail so that `egrep` does not eat the exit code.
+set -o pipefail
+
+mvn --batch-mode clean verify | egrep -v "(^\[INFO\] Download|^\[INFO\].*skipping)"
+
+# Run tests using App Engine local devserver.
+test_localhost() {
+  # Pin to a specific commit so that java-repo-tools can be updated
+  # independently of this repo.
+  git clone https://github.com/GoogleCloudPlatform/java-repo-tools.git
+  (
+  cd java-repo-tools
+  git checkout 74755a28f01aedf57992f473eb2d0201b1e04f2a
```

Comment:
I'd rather you used TOT and if it breaks everything, so be it, that's the way G3 mostly works and I think it's relatively sane.